src/utils/dialog/dialog.js

```diff
@@ -186,6 +189,38 @@ cwc.utils.Dialog.prototype.showTemplate = function(title, template, values) {
 
 
 /**
```

Comment:
Why move this out? You are going to expose details of dialog class to gcloud that it shouldn't know, i.e. this line:

`var buttonsDiv = goog.dom.getElement(this.prefix + 'buttons');`

If you change the id from `this.prefix + 'buttons` you will break gcloud since it has knowledge of internal implementation details that it shouldn't.